ai-services/cmd/ai-services/cmd/application/create.go

```diff
@@ -821,6 +821,10 @@ func constructPodDeployOptions(podAnnotations map[string]string) map[string]stri
 
 	// loop over each of the hostPortMappings to construct the 'publish' option
 	for containerPort, hostPort := range hostPortMappings {
+		if hostPort == "0" {
```

Comment:
wondering with above if block in the template, we can skip this here! not sure what is the best solution.